pagerduty/event_orchestration_path_test.go

```diff
@@ -515,7 +515,7 @@ func TestEventOrchestrationPathRouterUpdate(t *testing.T) {
 		if !reflect.DeepEqual(v.OrchestrationPath, input) {
 			t.Errorf("Request body = %+v, want %+v", v, input)
 		}
-		w.Write([]byte(`{"orchestration_path": { "type": "router", "parent": { "id": "E-ORC-1", "self": "https://api.pagerduty.com/event_orchestrations/E-ORC-1", "type": "event_orchestration_reference" }, "sets": [ { "id": "start", "rules": [ { "actions": { "dynamic_route_to": { "source": "event.summary", "regex": "service name: (.*)", "lookup_by": "service_name" }}, "conditions": [], "id": "E-ORC-DYNAMIC-RULE"}, { "actions": { "route_to": "P3ZQXDF" }, "conditions": [ { "expression": "event.summary matches part 'orca'" }, { "expression": "event.summary matches part 'humpback'" } ], "id": "E-ORC-RULE-1"}]}]}}`))
+		w.Write([]byte(`{"orchestration_path": { "type": "router", "parent": { "id": "E-ORC-1", "self": "https://api.pagerduty.com/event_orchestrations/E-ORC-1", "type": "event_orchestration_reference" }, "sets": [ { "id": "start", "rules": [ { "actions": { "escalation_policy": "POLICY"}}, {"actions": { "dynamic_route_to": { "source": "event.summary", "regex": "service name: (.*)", "lookup_by": "service_name" }}, "conditions": [], "id": "E-ORC-DYNAMIC-RULE"}, { "actions": { "route_to": "P3ZQXDF" }, "conditions": [ { "expression": "event.summary matches part 'orca'" }, { "expression": "event.summary matches part 'humpback'" } ], "id": "E-ORC-RULE-1"}]}]}}`))
```

Comment:
Can we move the EP action to a test for an orchestration path type that supports the action, e.g., `TestEventOrchestrationPathGlobalUpdate`? Just so it reflects a real life scenario.

Reply:
Oh yeah, I overlooked this. I shoehorned it in for testing and didn't come back to write a specific test. thank you for calling it out

Reply:
I think it's ok to shoehorn it into the existing test, but we can also keep it as a separate test that you added in https://github.com/alexzakabluk/go-pagerduty/pull/3/commits/f3f7697777d1b6ba65863faf66497ac4187dc819. I was mostly talking about using an orchestration type other than the router, because in real life routers don't support this action, only Global and Service paths do. That's why I suggested throwing it into the payload in `TestEventOrchestrationPathGlobalUpdate` since it already covers all other actions.